lgc/builder/InOutBuilder.cpp

```diff
@@ -1258,6 +1258,7 @@ unsigned InOutBuilder::getBuiltInValidMask(BuiltInKind builtIn, bool isOutput) {
   enum class StageValidMask : unsigned {
     C = (1 << ShaderStageCompute),
     D = (1 << ShaderStageTessEval),
+    DP = (1 << ShaderStageTessEval) | (1 << ShaderStageFragment),
```

Comment:
Actually, gl_ViewIndex could act as a built-in input in all traditional shader stages. See the spec: https://github.com/KhronosGroup/GLSL/blob/master/extensions/ext/GL_EXT_multiview.txt

Reply:
```
The input variable gl_ViewIndex is available in vertex, tessellation,
geometry, and fragment shading languages and corresponds to the
ViewIndex SPIR-V input.
```

Reply:
Thanks Rex; I will fix it to do that.